source/CurieBleSerial.cpp

```diff
@@ -0,0 +1,378 @@
+/*
```

Comment:
We should be able to inline. I removed this file and the DfRobotBleSerial file from the project, but forgot to yank them from this version in the source control. Will fix with next iteration.